core/control-plane/contract-core/src/main/java/org/eclipse/dataspaceconnector/contract/ContractServiceExtension.java

```diff
@@ -71,7 +71,9 @@
 @CoreExtension
 public class ContractServiceExtension implements ServiceExtension {
 
-    private static final long DEFAULT_ITERATION_WAIT = 5000; // millis
+    private static final int DEFAULT_ITERATION_WAIT = 5000;
+    @Setting(value = "the iteration wait time in milliseconds on the state machine. Default value 5000")
```

Comment:
Can put a space here for clarity? The others should be spaced as well if you want to go ahead and clean that up.